org.angryip.ipscan.yaml

```diff
@@ -0,0 +1,81 @@
+app-id: org.angryip.ipscan
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: ipscan
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=xdg-config/gtk-3.0:ro;
+  - --persist=.swt
+  - --persist=.java
+  - --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+  - --env=JAVA_HOME=/app/jre
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+
+  - name: ipscan
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 ipscan.jar -t /app/lib/ipscan
+      - install -Dm755 ipscan -t /app/bin
+      - install -Dm644 org.angryip.ipscan.desktop -t /app/share/applications
+      - install -Dm644 org.angryip.ipscan.metainfo.xml -t /app/share/metainfo
+      - install -Dm644 org.angryip.ipscan.svg -t /app/share/icons/hicolor/scalable/apps
+      - install -Dm644 org.angryip.ipscan32.png /app/share/icons/hicolor/32x32/apps/org.angryip.ipscan.png
+      - install -Dm644 org.angryip.ipscan48.png /app/share/icons/hicolor/48x48/apps/org.angryip.ipscan.png
+      - install -Dm644 org.angryip.ipscan128.png /app/share/icons/hicolor/128x128/apps/org.angryip.ipscan.png
+      - install -Dm644 org.angryip.ipscan256.png /app/share/icons/hicolor/256x256/apps/org.angryip.ipscan.png
+
+    sources:
+      - type: file
+        url: https://github.com/angryip/ipscan/releases/download/3.9.1/ipscan-linux64-3.9.1.jar
+        sha256: b8b12628c324cddb1e1a464c1caf2597b66ce8f5f1057ffa86c1fe7b1c241b40
+        dest-filename: ipscan.jar
+        only-arches: [x86_64]
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/angryip/ipscan/releases/latest
+          version-query: .tag_name
+          url-query: .assets[] | select(.name=="ipscan-linux64-" + $version + ".jar") | .browser_download_url
+
+      - type: file
+        path: ipscan
+
+      - type: file
+        path: org.angryip.ipscan.desktop
+
+      - type: file
+        path: org.angryip.ipscan.metainfo.xml
+
+      - type: file
+        url: https://raw.githubusercontent.com/angryip/ipscan/master/resources/images/icon.svg
```

Comment:
these should be from a commit not `master` otherwise shasum will change breaking the build

Same for everything below.